src/Mealz/MealBundle/Entity/Participant.php

```diff
@@ -39,11 +38,11 @@ class Participant
     private ?Slot $slot = null;
 
     /**
-     * @ORM\ManyToMany(targetEntity="Dish", inversedBy="participants")
+     * @ORM\ManyToMany(targetEntity="Dish")
      *
-     * @var Collection<int, Dish>
+     * @var Collection<int, Dish>|null
      */
-    private Collection $combinedDishes;
+    private ?Collection $combinedDishes;
```

Comment:
I don't see any reason why it can be null? In the constructor, it is initialized with an empty array (not null). In the setter it's cleared and after that, it'll be overriden. When will it be null?

Reply:
Doctrine doesn't use normal object instantiation for entities, i.e. reflection. This means no constructor execution.

A normal meal participation will have null for combinedDishes.

Reply:
I see.